### PR TITLE
Changed CBSN logos to match

### LINKED ISSUE #3714: Add: [adult swim]Canada
**Channel Name:** [adult swim]Canada(24/7 Channel)
**Country:** Canada
**Language:** English
**Website:** xxx
**Stream URL (optional):** http://31.220.41.88:8081/live/us-cartoon.stream/playlist.m3u8?token=null
**Notes (optional):** Link said Cartoon Network US, But It Looks Like Shows [adult swim]Canada. 
Working in Pot Player and GSE Smart Player

### PR DESCRIPTION
**Pull request by jkemp814**
Switched out logos, all were set to cbs news. Now localized.
Also closes https://github.com/iptv-org/iptv/issues/3714

### CODE REVIEW

**Comment by jkemp814** [comment]
So far from what I have checked most of those channels recently added with Latinoamérica in the title are not loading. ESPN, Disney Junior, AMC, AXN, Boomerang, and more. Most of them have http://209.91.213.10:8088/play/ as the address. Maybe the server down.

**Comment by jkemp814** [comment]
These logos are redirecting to a site called DooMovies: 
https://www.img09.xyz/assets/img/ch_logo/hd-foxsports1.png
https://www.img09.xyz/assets/img/ch_logo/hd-foxsports2.png
https://www.img09.xyz/assets/img/ch_logo/hd-foxsports3.png

When checking them in the browser. They are not showing up in Kodi, or IPTVnator.

**Comment by Dum4G** [comment]
Don't play with the imgur link format. Either it makes a redirect to an i.imgur format anyways or makes it load imgur page for the image, not the image itself

**Comment by jkemp814** [comment]
You are referring with the logos I put up? or the ones listed for FOX Sports?

**Comment by Dum4G** [comment]
The one you put up. If https://www.img09.xyz act wrong feel free to replace them

**Comment by jkemp814** [comment]
I didn't put those logo links up. I see all the links I put up in Kodi. Do you remember which ones redirect I put up?

**Comment by jkemp814** [comment]
I wasn't going to change those logos because the  Latinoamérica channels are not working. Was going to wait. Maybe something with their server.

**Comment by Dum4G** [comment]
If you don't want to search small icon replacements, here's where I found both icons and channels. Beware, that the website is infested with porn and casino banners, it's literally 80% of it's visible content. 
https://www.doomovie-hd.com/?r=live_tv

**Comment by Dum4G** [comment]
> I didn't put those logo links up. I see all the links I put up in Kodi. Do you remember which ones redirect I put up?

https://imgur.com/zeg2ZGW for example

**Comment by Dum4G** [comment]
There's no need to remove i. from imgur links, if you check in your browser those will be redirected anyway

**Comment by jkemp814** [comment]
> There's no need to remove i. from imgur links, if you check in your browser those will be redirected anyway

I using the link that imgur provide. Sometime the server will do that. I'm not removing it. All the logos I posted are showing up.

**Comment by jkemp814** [comment]
Should I add the i.imgur.com?

**Comment by Dum4G** [comment]
> Should I add the i.imgur.com?

If you don't mind, please

**Comment by Dum4G** [comment]
Can we have that link added in this PR? Thanks
#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US;CA" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Entertainment",Adult Swim 24/7
http://31.220.41.88:8081/live/us-cartoon.stream/playlist.m3u8?token=null

**Comment by Dum4G** [comment]
Is it time to approve this PR?

**Comment by jkemp814** [comment]
Yes, please

File: channels/us.m3u [hunk 1]
```diff
@@ -5493,7 +5493,7 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxsamuelgoldwyn/CDN/playlist.
 https://samuelgoldwyn-films-roku.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="SamuelGoldwynClassics.us" tvg-name="Samuel Goldwyn Classics" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/oCjZS0v.jpg" group-title="Classic",Samuel Goldwyn Classics
 https://samuelgoldwyn-classic-roku.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="SamuraiJack.us" tvg-name="Samurai Jack (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="",Samurai Jack (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="SamuraiJack.us" tvg-name="Samurai Jack (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zIjbwSa.jpg" group-title="Series",Samurai Jack (Adult Swim 24/7 Stream) [Geo-blocked]
```
Comment (Dum4G) [inline channels/us.m3u:5496]: I think all of those series from Adult Swim fall into Animation category

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u [hunk 2]
```diff
@@ -151,17 +151,17 @@ https://a.jsrdn.com/broadcast/542cb2ce3c/+0000/c.m3u8
 http://31.220.41.88:8081/live/us-cinemaxaction.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="LocalNowAdrenalineRush.us" tvg-name="Local Now Adrenaline Rush" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/601341747e82bb6a262f276b" group-title="Entertainment",Adrenaline Rush
 https://linear-133.frequency.stream/dist/localnow/133/hls/master/playlist.m3u8
-#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Entertainment",Adult Swim (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Animation",Adult Swim (1080p) [Not 24/7]
```
Comment (Dum4G) [inline channels/us.m3u:154]: Adult Swim as a channel itself shows variety of content. Entertainment was right category

File: channels/us.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/us.m3u:5496]: Okay I change those.

File: channels/us.m3u [hunk 3]
```diff
@@ -412,7 +412,7 @@ https://thegateway.app/BizAndYou/Biz_480p/playlist.m3u8
 https://thegateway.app/BizAndYou/Biz_1080p/playlist.m3u8
 #EXTINF:-1 tvg-id="BizTVHD.us" tvg-name="Biz TV HD" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/4pEmcFH.png" group-title="Lifestyle",Biz TV HD (720p)
 https://thegateway.app/BizAndYou/Biz_720p/playlist.m3u8
-#EXTINF:-1 tvg-id="BlackJesus.us" tvg-name="Black Jesus (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/2/2b/Black_Jesus_title_card.png" group-title="",Black Jesus (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="BlackJesus.us" tvg-name="Black Jesus (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/2/2b/Black_Jesus_title_card.png" group-title="Animation",Black Jesus (Adult Swim 24/7 Stream) [Geo-blocked]
```
Comment (Dum4G) [inline channels/us.m3u:415]: That one is truly a Series with casted actors

File: channels/us.m3u [hunk 4]
```diff
@@ -786,7 +786,7 @@ https://playout4multirtmp.tulix.tv/live8/Stream1/playlist.m3u8
 https://granicusliveus4-a.akamaihd.net/cerritos/G0010_002/playlist.m3u8
 #EXTINF:-1 tvg-id="CGTV.us" tvg-name="CGTV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/f6dfLXA.png" group-title="Legislative",CGTV
 https://reflect-live-champaign.cablecast.tv/live/CELL-296k-234p/CELL-296k-234p.m3u8
-#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Channel 5 (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/sCRPkPk.jpg" group-title="Series",Channel 5 (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Channel 5 (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/sCRPkPk.jpg" group-title="Animation",Channel 5 (Adult Swim 24/7 Stream) [Geo-blocked]
```
Comment (Dum4G) [inline channels/us.m3u:789]: Adult Swim as a channel itself shows variety of content. Entertainment was right category

File: channels/us.m3u [hunk 5]
```diff
@@ -6590,5 +6590,5 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
 https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-name="Канал Disney" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p)
 http://188.40.68.167/russia/disney/playlist.m3u8
-#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US;CA" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Entertainment",Adult Swim
+#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US;CA" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Animation",Adult Swim
```
Comment (Dum4G) [inline channels/us.m3u:6593]: Adult Swim as a channel itself shows variety of content. Entertainment was right category

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**